src/SDK/SmartDetectorsShared/Clients/TelemetryDbType.cs

```diff
@@ -0,0 +1,28 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="TelemetryDbType.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.RuntimeEnvironment.Contracts
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+
+    /// <summary>
+    /// An enum for the type of DB on which to run queries that are part of alerts presentation
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum TelemetryDbType
```

Comment:
>TelemetryDbType [](start = 16, length = 15)

huh? please remove

Reply:
pass a string to the TelemetryDbType class ctor and use it for telemetry

---
In reply to: [257923834](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/49#discussion_r257923834) [](ancestors = 257923834)